evalai/utils/common.py

```diff
@@ -67,3 +67,4 @@ def clean_data(data):
     data = BeautifulSoup(data, "lxml").text.strip()
     data = ' '.join(data.split())
     data.encode("utf-8")
```

Comment:
`encode` returns the new encoded lists list, it doesn't modify the original. So the changes doesn't stick.

Reply:
@isht3 Yes 😅. Fixed. Thanks for pointing that out.